.github/workflows/ci-cd.yml

```
@@ -1,6 +1,9 @@
 ---
 name: CI/CD
 
+permissions:
+  contents: read
```

It's even better to do
```suggestion
permissions: {}
```
globally and add granular `permissions` settings to each job to follow the principle of least privileges closer.